### DISCUSSION TITLE
Modifier les informations d'un service existant

### DISCUSSION THREAD

**Discussion by cedricn1974** [AmauriC/tarteaucitron.js#697] [Q&A]
Bonjour,
Est-il possible de modifier certaines informations d'un service déjà existant (type,name, uri...)  sans avoir à toucher le fichier tarteaucitron.services.js ou créer un service custom basé sur le service existant ?
Merci d'avance !
Cédric

**Comment by AmauriC** [comment]
Il n'y a pas d'option prévu pour modifier ces valeurs.
C'est pour une correction ou quelque chose de spécifique à vos sites?

**Reply by cedricn1974** [reply]
Merci de votre réponse.
C'est pour quelque chose de spécifique à notre site, suite à des demandes du service juridique. Par exemple, sur les URLs des politiques de confidentialité des services.
